### DISCUSSION TITLE
Plausible's vendor_id for Paddle is hardcoded.

### DISCUSSION THREAD

**Discussion by ACPK** [plausible/analytics#627] [Self-Hosted Support]
I will make a PR, but Plausible's vendor_id (49430) is hardcoded in config/config.exs on line 82.

````
config :plausible, :paddle,
  vendor_id: "49430",
  vendor_auth_code: System.get_env("PADDLE_VENDOR_AUTH_CODE")
````

**Comment by metmarkosaric** [comment]
Moving this to discussions as it's not really an issue. It works the way it is intended. This section of the codebase is not really relevant to self-hosters, it is for our cloud.
